AdminUI/src/app/app-routing.module.ts

```diff
@@ -13,7 +13,7 @@ const routes: Routes = [
     { path: 'identities', component: IdentityListComponent },
     { path: 'tiers', component: TierListComponent },
     { path: 'tiers/create', component: TierEditComponent },
-    { path: 'tiers/:id', component: TierEditComponent },
+    { path: 'tiers/:id/:name', component: TierEditComponent },
```

Comment:
This route looks weird. Why is there id AND name?

Reply:
Yeah I agree it looks weird, but currently, we have no endpoint to get a single tier. By routing like this, we can at least create a tier object with both the Id and the name, otherwise, the edit tier page would have to display an empty name.

Reply:
Why don't you just implement the getById method of the service so that it fetches all Tiers and then filters them for the one with the id we are looking for?